### PR TITLE
fix: use updateProductVariantPrices mutation to save variant price

### LINKED ISSUE #170: [v3.0.0] Cannot set price for variants
## Issue Description
Setting and publishing the price for a variant does not take effect.

No errors are reported. This is different than #91
Viewing the product in a storefront will also show the product without a price.

## Steps to Reproduce
Please provide starting context, i.e. logged in as a user, configure a particular payment method.
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is reset to zero.

## Possible Solution
Not obligatory, but suggest a fix/reason for the bug

### Versions
release-3.0.0

### PR DESCRIPTION
**Pull request by trojanh**
Signed-off-by: trojanh <rajan.tiwari@kiprosh.com>

Resolves https://github.com/reactioncommerce/reaction-admin/issues/170
Impact: **breaking|major**  
Type: **bugfix**

## Issue
Unable to set product variant price
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is reset to zero.

## Solution
Use  `updateProductVariantPrices` mutation to set the price.

## Breaking changes
none

## Testing
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is NOT reset to zero.
4. Also publish and refresh the variant to see it the changes get saved.

variant-price-update

### CODE REVIEW

**Comment by spencern** [comment]
Thanks for the PR!

File: imports/plugins/included/product-admin/client/hocs/withVariantForm.js [hunk 1]
```diff
@@ -189,6 +199,28 @@ const wrapComponent = (Comp) => (
...
+              shopId,
+              variantId: opaqueVariantId,
+              prices: {
+                [fieldName]: value
+              }
+            }
+          }
+        });
+      } catch (error) {
+        Alerts.toast(i18next.t("productDetailEdit.updateProductFieldFail", { [fieldName]: value }), "error");
```
Comment (willopez) [inline imports/plugins/included/product-admin/client/hocs/withVariantForm.js]: For new code we want to use the new notification library `notistack` as seen here: https://github.com/reactioncommerce/reaction-admin/blob/e9cc890e3a1b63cec0804b260a161ed60f685a79/imports/plugins/core/taxes/client/containers/GeneralTaxSettings.js#L31 Please update to use it.

**Review by willopez — CHANGES_REQUESTED** [review]
One small change necesary, otherwise looks good.

**Comment by manueldelreal** [comment]
Save for @willopez review comment and the linting errors on the checks this one does permit the price update via the admin ui.

**Comment by trojanh** [comment]
@willopez I have replaced all occurrences of `Alerts.alert` and `Alert.toast` to use `snackbar`. Also, fixed the linting error and rebased the branch. Please review the changes.

**Comment by willopez** [comment]
thanks @trojanh!

**Pull request merged**